### PR TITLE
Add filters

### PR DESCRIPTION
**Pull request by nkTribe**
Copy over `filters.js` file from Ambassador and modify the functions so they work and don't rely on `tribe.util`. Also imported `moment.js`

### CODE REVIEW

**Comment by henryachen** [comment]
Only add functions that you need.

File: src/filters/filters.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+     */
+    momentDateAbbrLong(momentObj) {
+      return momentObj.format('MMM YYYY');
+    },
+    /**
+     * @param emv {number}
+     * @param sponsoredEmv {number}
+     * @returns {number} organic emv
+     */
+    organicEmv(emv, sponsoredEmv) {
```
Comment (henryachen) [inline src/filters/filters.js]: this is just a subtraction function

File: src/filters/filters.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+     */
+    emvFromMicros(value) {
+      return Number(value / 1e6).toFixed(0);
+    },
+    /**
+     * @param emv {number} emv for one brand
+     * @param totalEmv {number} total emv for subset of brands
+     * @returns {string} emv share (emv divided by the total emv for a subset of brands)
+     */
+    emvShare(emv, totalEmv) {
```
Comment (henryachen) [inline src/filters/filters.js]: this is just a percentage function?

**Comment by nkTribe** [comment]
This may not be needed after all, so I'm closing the pull request for the moment.

**Pull request closed without merging**